native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.ios.js

```diff
@@ -38,9 +33,17 @@ import {
   spacingSm,
   spacingXl,
 } from 'bpk-tokens/tokens/base.react.native';
+
+import React from 'react';
```

Comment:
I would keep roughly the order from before

Reply:
or is this a prettier change?

Reply:
Don't think is a prettier change, I probably moved them